.azure-pipelines/baseline_test/baseline.test.template.yml

```diff
@@ -1,194 +1,76 @@
 jobs:
   - job: get_impacted_area
+    cancelTimeoutInMinutes: 10
     displayName: "Get impacted area"
     timeoutInMinutes: 240
     continueOnError: false
     pool: sonic-ubuntu-1c
     steps:
       - template: ../impacted_area_testing/get-impacted-area.yml
 
-  - job: impacted_area_t0_elastictest
-    displayName: "impacted-area-kvmtest-t0 by Elastictest"
+  - job: run_impacted_area_test
+    cancelTimeoutInMinutes: 10
```

Comment:
It means, the timeout for running time is 10 minutes?

Reply:
I saw that the baseline tests don't quickly respond to the user's cancel request. They run for hours after the user clicks on "cancel" pipeline. This timeout value is for how long azure will wait before it performs force cancel of all the running jobs after the user selects cancel. I believe this will still not be sufficient as we interact with elastictest. I hope to address that in a different PR.